### DISCUSSION TITLE
Achievements and Progression

### DISCUSSION THREAD

**Discussion by tcarlsen** [CarlsnOP/moba-manager#29] [Ideas]
Achievements can provide players with goals and rewards beyond the core gameplay. What kinds of achievements should we include in MOBA Manager? How can we structure progression to keep players motivated? Share your ideas on how to make achievements fun and rewarding.

**Comment by CarlsnOP** [comment]
When a hero reaches max level (99) he can ascend going from common -> uncommon -> epic -> rare -> legendary
Unlocking more skill slots and higher growth per level
